content/en/docs/instrumentation/net/automatic/config.md

```diff
@@ -2,8 +2,8 @@
 title: Configuration and settings
 linkTitle: Configuration
 weight: 20
-cSpell:ignore: NETFX AZUREAPPSERVICE UNHANDLEDEXCEPTION CORECLR CLSID dylib
-Bitness
+# prettier-ignore
+cSpell:ignore: NETFX AZUREAPPSERVICE UNHANDLEDEXCEPTION CORECLR CLSID dylib Bitness bitness BITNESS
```

Comment:
for `Bitness bitness BITNESS` only bitness should do the job afaik

Reply:
Right. A remnant of my attempts at understanding what was the error about...

Reply:
:D